mapomatic/layouts.py

```diff
@@ -39,13 +39,18 @@ def matching_layouts(circ, cmap, strict_direction=False, call_limit=10000):
 
     Parameters:
         circ (QuantumCircuit): Input quantum circuit
-        cmap (list): Coupling map
+        cmap (list or IBMQBackend): Coupling map or backend instance
```

Comment:
Why does this have to be an IBMQ backend? Should it just be any `BaseBackend` or `BackendV1`